presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestReorderJoins.java

```diff
@@ -98,7 +98,7 @@ public void testEliminateCrossJoinWithNonEqualityCondition()
                                 anyTree(
                                         join(INNER, ImmutableList.of(equiJoinClause("P_PARTKEY", "L_PARTKEY")), Optional.of("P_NAME < cast(L_COMMENT AS varchar(55))"),
                                                 anyTree(PART_WITH_NAME_TABLESCAN),
-                                                anyTree(filter("L_PARTKEY <> L_ORDERKEY", LINEITEM_WITH_COMMENT_TABLESCAN)))),
+                                                anyTree(filter("L_PARTKEY <> L_ORDERKEY AND L_ORDERKEY = \"$INTERNAL$DEFERRED_SYMBOL_REFERENCE\"(\"128\", \"dynamic_filter_orderkey\")", LINEITEM_WITH_COMMENT_TABLESCAN)))),
```

Comment:
I'm going to change it to `L_ORDERKEY = DYNAMIC_FILTER_REFERENCE({join_node_id = 1, symbol = 'symbol_provided_by_the_build_side', id = 'dynamic_filter_symbol_provided_by_the_build_side'})`. Also i'm going to print ids for every join node, so everyone reading the explain can link the filters together.